stream/src/main/scala/org/apache/pekko/stream/scaladsl/Flow.scala

```diff
@@ -181,7 +182,22 @@ final class Flow[-In, +Out, +Mat](
    * '''Cancels when''' original [[Flow]] cancels
    */
   def contramap[In2](f: In2 => In): Flow[In2, Out, Mat] =
-    Flow.fromFunction(f).viaMat(this)(Keep.right)
+    Flow.fromFunction(f).viaMat(this)(Keep.right).withAttributes(DefaultAttributes.contramap)
+
+  /**
+   * Transform this Flow by applying a function `f` to each *incoming* upstream element before
+   * it is passed to the [[Flow]], and a function `g` to each *outgoing* downstream element.
+   *
+   * '''Emits when''' the mapping function `g` returns an element
+   *
+   * '''Backpressures when''' original [[Flow]] backpressures
+   *
+   * '''Completes when''' original [[Flow]] completes
+   *
+   * '''Cancels when''' original [[Flow]] cancels
```

Comment:
again, please add `@since` to new methods/functions

Reply:
I always forget this ,sorry